pkg/kubelet/cm/dra/manager_test.go

```diff
@@ -335,6 +339,41 @@ func genTestPod() *v1.Pod {
 	}
 }
 
+// genTestPodWithExtendedResource generates pod object
+func genTestPodWithExtendedResource() *v1.Pod {
+	return &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      podName,
+			Namespace: namespace,
+			UID:       podUID,
+		},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name: containerName,
+					Resources: v1.ResourceRequirements{
+						Requests: v1.ResourceList{
+							"example.com/gpu": resource.MustParse("1"),
```

Comment:
Do we need any tests here with:
1) mixed regular and extended resources
2) multiple extended resources

Or are those covered elsewhere for the kubelet side?